app/definitions/IRocketChatRecord.ts

```diff
@@ -1,4 +1,4 @@
 export interface IRocketChatRecord {
-	id: string;
+	_id: string;
```

Comment:
Are we using this for our local db or for data coming from the server?
The server will always return `_id`, because it's mongodb.
The local db will always convert it to `id`, because it's sqlite.